packaging/conda/build.sh

```diff
@@ -3,29 +3,21 @@
 # Set-up the shell to behave more like a general-purpose programming language
 set -euo pipefail
 
-# Choose the build preset for macOS/Linux
-if [[ "$(uname)" == "Linux" ]]
-then
-  CMAKE_PRESET="linux-gcc-release"
-else
-  CMAKE_PRESET="macos-clang-release"
-fi
-
 # Configure project
-cmake --fresh --preset $CMAKE_PRESET -DBUILD_JARS=OFF -DTESTING=OFF
+cmake -B build-conda -S . -D BUILD_JARS=OFF -D TESTING=OFF -D CMAKE_BUILD_TYPE=Release -G Ninja
 
 # Build MODL and MODL_Coclustering
-cmake --build --preset $CMAKE_PRESET --parallel \
+cmake --build build-conda --parallel \
   --target MODL MODL_Coclustering KhiopsNativeInterface KNITransfer _khiopsgetprocnumber
 
 # Move the binaries to the Conda PREFIX path
-mv ./build/$CMAKE_PRESET/bin/MODL* "$PREFIX/bin"
-mv ./build/$CMAKE_PRESET/bin/_khiopsgetprocnumber* "$PREFIX/bin"
-mv ./build/$CMAKE_PRESET/bin/KNITransfer* "$PREFIX/bin"
-mv ./build/$CMAKE_PRESET/lib/libKhiopsNativeInterface* "$PREFIX/lib"
+mv ./build-conda/bin/MODL* "$PREFIX/bin"
```

Comment:
What I had in mind was rather `./build/conda`; but `./build-conda` can also do, because there is no confusion with the `conda-build` / `conda build` command anymore.

Reply:
Done with `build/conda`.